packages/core-js/src/types/gateway/authenticate.ts

```diff
@@ -6,6 +6,10 @@ export type AuthData =
   | {
       authToken: string;
       provider: 'okto';
+    }
+  | {
+      idToken: string;
+      provider: 'okto';
     };
```

Comment:
@karansinglacoindcx what is the difference b/w 2nd and 3rd ?

Reply:
@karansinglacoindcx remove {
      authToken: string;
      provider: 'okto';
    } is not needed